bower.json

```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "polymer": "*",
     "palindrom-bunny": "Palindrom/palindrom-bunny",
-    "dom-bind-notifier": "*"
+    "dom-bind-notifier": "*",
+    "fast-json-patch": ">=1.1.0"
```

Comment:
Again, can't we use `fast-json-patch` from `node_modules`?

Also `dom-bind-notifer` should be removed while we're at it.